engine/orchestration/src/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -3685,7 +3685,7 @@ public void setStoragePoolAllocators(final List<StoragePoolAllocator> storagePoo
     //
 
     @MessageHandler(topic = Topics.VM_POWER_STATE)
-    private void HandlePowerStateReport(final String subject, final String senderAddress, final Object args) {
+    protected void HandlePowerStateReport(final String subject, final String senderAddress, final Object args) {
```

Comment:
@sureshanaparti can you explain in detail how you got the exception and how this fix fixes that